src/pages/Admin/Charity/Dashboard/Dashboard.tsx

```diff
@@ -12,7 +12,7 @@ export default function Dashboard() {
 
   return (
     <div className="@container w-full max-w-4xl grid content-start mt-6">
-      <Seo title="Charity Dashboard" />
+      <Seo title="Nonprofit Dashboard" />
```

Comment:
@SovereignAndrey are we dropping the term "charity" completely?
If so, it would be prudent to update our internal variable/field names from `*charity*` to `*nonprofit*` to avoid confusion later (e.g. `charityName -> nonprofitName`, `CHARITY_LINKS -> NONPROFIT_LINKS` etc.)

Reply:
we could change everything in frontend, but would eventually come into inconsistency with backend as we can't change db `attribute` names easily

Reply:
As I understand, yes, we're moving to the more inclusive "nonprofit" term. We can and should move to change internal vars and such too, but it's def a low priority thing (esp with the issues we'll have changing the DB fields as Justin mentioned. Wen Postgres... :sweat: ). :+1: